Framework/DataHandling/src/SaveNexusProcessed.cpp

```diff
@@ -331,21 +331,29 @@ void SaveNexusProcessed::appendEventListData(std::vector<T> events,
   if (events.empty())
     return;
 
-  typename std::vector<T>::const_iterator it;
-  typename std::vector<T>::const_iterator it_end = events.end();
-  size_t i = offset;
+  auto it = events.cbegin();
```

Comment:
Should this be `const auto`? I'm suspicious that the compiler will figure it out from the `cbegin()`.

Reply:
This shouldn't matter (`events` is const and `.cbegin()` and `.cend()` returns a const_interator), but never hurts to be explicit.